src/openlcb/SimpleStack.cxx

```diff
@@ -157,7 +162,8 @@ void SimpleStackBase::default_start_node()
 #if OPENMRN_HAVE_POSIX_FD
     if (CONFIG_FILENAME != nullptr)
     {
-        auto *space = new FileMemorySpace(CONFIG_FILENAME, CONFIG_FILE_SIZE);
+        auto *space =
+            new FileMemorySpace(configUpdateFlow_.get_fd(), CONFIG_FILE_SIZE);
         memory_config_handler()->registry()->insert(
             node(), openlcb::MemoryConfigDefs::SPACE_CONFIG, space);
         additionalComponents_.emplace_back(space);
```

Comment:
Please correct the `#endif` line below to have `OPENMRN_HAVE_POSIX_FD` instead of `NOT ARDUINO, YES ESP32` since we moved to a new define.